src/custom/utils/operator.ts

```diff
@@ -13,8 +14,8 @@ const API_BASE_URL: Partial<Record<ChainId, string>> = {
 }
 
 const DEFAULT_HEADERS = {
-  'Content-Type': 'application/json'
-  // TODO: Maybe add a custom header for the AppId (same as the signing tx)
+  'Content-Type': 'application/json',
+  'X-AppId': APP_ID.toString()
```

Comment:
Server expects this?

Reply:
now it does :)

Reply:
https://github.com/gnosis/gp-v2-services/pull/135